enhancements/olm-admission-webhooks.md

```diff
@@ -0,0 +1,135 @@
+---
+title: olm-admission-webhook-support
+authors:
+  - "@awgreene"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2020-03-12
+last-updated: 2020-03-12
+status: implementable
+---
+
+# OLM Admission Webhook Support
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement covers the necessary steps to add the capability for OLM to support [validating](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#validatingadmissionwebhook) and [mutating](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#mutatingadmissionwebhook) admission webhooks.
+
+## Motivation
+
+More and more Operators are shipping webhooks to deliver on core parts of their functionality. Today, operators shipped with OLM are required to manually create and manage their webhooks independently of OLM due to lack of support. In this process, each operator re-creates the logic to implement a webhook and the logic to rotate and manage the coresponding certificates. OLM can help automate much of this process and provide a first-class experience to users, further strengthen OLM's value proposition.
+
+### Goals
+
+- Operator authors can include validating and mutating admission webhooks in their bundle as a part of a CSV.
+- Operator author will not need to manage cert rotation for webhooks shipped with their operators.
+- Via the SubscriptionConfig field in the Subscription, Cluster Admins will be able to integrate with tools like [service-ca operator](https://github.com/openshift/service-ca-operator) or [cert-manager](https://github.com/jetstack/cert-manager) projects by:
```

Comment:
I would remove the implementation detail (subscripction config) from the goals

Reply:
Addressed in the latest version of the PR.